### PR TITLE
Added song input to to form settings

### PR DESCRIPTION
**Pull request by benl95**

### CODE REVIEW

File: server/components/form-settings/form-settings.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 {% call fieldset({ legend: 'Main personal information', sronly: true }) %}
 
-  {{ input({
```
Comment (RobinStut) [inline server/components/form-settings/form-settings.html:27]: something went wrong with the code formatting. There are changes with the indenting

File: server/components/form-settings/form-settings.html [hunk 2]
```diff
@@ -150,37 +150,47 @@
...
+    hook: 'genre-form',
     class: 'form__item--is-visible'
   }) %}
 
-    {{ radio({
-      name: 'level',
-      value: 'Noob',
-      image: '/assets/images/noob.png'
-    }) }}
+  <option name='genre' value='deephouse'>Deephouse</option>
```
Comment (RobinStut) [inline server/components/form-settings/form-settings.html:175]: Maybe we can write a loop over all genres

Comment (Mennauu) [reply server/components/form-settings/form-settings.html:175]: Good idea, we should do this! Also create a option form element macro

File: server/components/form-settings/form-settings.html (hunk 1 above)
Comment (benl95) [reply server/components/form-settings/form-settings.html:27]: Fixed it, could you check if it is good now?

**Comment by Mennauu** [comment]
- Created a macro for `<option>`
- Created a genre data file
- <option> are generated based on data from genre data file (using a for loop in Nunjucks)
- Adjusted the overal form to work again when JavaScript is enabled

File: server/components/form-settings/form-settings.js [hunk 3]
```diff
@@ -115,6 +114,8 @@ class FormSettings {
   formHandler() {
     this.closeNotification()
 
+    const disabledArray = [1, 6, 7]
```
Comment (RobinStut) [inline server/components/form-settings/form-settings.js:117]: What is the function of this "disabledArray"? just curious

Comment (chelseadoeleman) [reply server/components/form-settings/form-settings.js:117]: I was wondering about that too

**Pull request merged**